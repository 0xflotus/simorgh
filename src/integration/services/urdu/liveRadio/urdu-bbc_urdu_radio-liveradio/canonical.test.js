/**
 * WARNING: DO NOT EDIT. This file is automatically generated by src/integration/utils/runTests/writeTestFiles/writeTestFile.js
 * @service urdu
 * @pathname /urdu/bbc_urdu_radio/liveradio
 */

import runCanonicalTests from '../../../../pages/liveRadio/canonicalTests';

describe('Canonical urdu liveRadio', runCanonicalTests);
