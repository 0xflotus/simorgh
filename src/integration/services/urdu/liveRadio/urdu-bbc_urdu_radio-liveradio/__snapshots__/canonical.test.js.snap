// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Canonical urdu liveRadio A11y I can see the skip to content link 1`] = `"مواد پر جائیں"`;

exports[`Canonical urdu liveRadio Analytics ATI 1`] = `"<img height=\\"1px\\" width=\\"1px\\" alt=\\"\\" style=\\"position:absolute\\" src=\\"https://logws1363.ati-host.net?s=598343&amp;s2=95&amp;p=urdu.bbc_urdu_radio.liveradio.page&amp;x1=[urn:bbc:ares:ws_media:live:bbc_urdu_radio]&amp;x2=[responsive]&amp;x3=[news-urdu]&amp;x4=[ur]&amp;x7=[player-live]&amp;x8=[simorgh-nojs]&amp;x9=[بی+بی+سی+اردو+ریڈیو+-+BBC+News+اردو]\\"/>"`;

exports[`Canonical urdu liveRadio Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/urdu"`;

exports[`Canonical urdu liveRadio Footer I can see the footer copyright and external linking text 1`] = `"© 2020 بی بی سی. بی بی سی بیرونی ویب سائٹس کے مواد کا ذمہ دار نہیں بیرونی لنکس کے بارے میں ہماری پالیسی."`;

exports[`Canonical urdu liveRadio Header I can see the branding 1`] = `"BBC News, اردو"`;

exports[`Canonical urdu liveRadio I can see the headline 1`] = `"بی بی سی اردو ریڈیو"`;

exports[`Canonical urdu liveRadio I can see the summary 1`] = `"اردو زبان کے معتبر ترین خبررساں اداروں میں ایک، بی بی سی اردو کی جانب سے مصدقہ خبریں اور جامع تجزیے"`;

exports[`Canonical urdu liveRadio Media player embed A11y I can read the media player title 1`] = `"Audio player"`;

exports[`Canonical urdu liveRadio Media player embed I can see the media player embed 1`] = `"https://polling.test.bbc.co.uk/ws/av-embeds/media/bbc_urdu_radio/liveradio/ur?morph_env=live"`;

exports[`Canonical urdu liveRadio SEO Canonical link 1`] = `"http://localhost:7080/urdu/bbc_urdu_radio/liveradio"`;

exports[`Canonical urdu liveRadio SEO Dir attribute 1`] = `"rtl"`;

exports[`Canonical urdu liveRadio SEO FB admins 1`] = `"100004154058350"`;

exports[`Canonical urdu liveRadio SEO FB app ID 1`] = `"1609039196070050"`;

exports[`Canonical urdu liveRadio SEO Heading level 1 1`] = `"بی بی سی اردو ریڈیو"`;

exports[`Canonical urdu liveRadio SEO Lang attribute 1`] = `"ur"`;

exports[`Canonical urdu liveRadio SEO Linked data 1`] = `
Object {
  "@context": "http://schema.org",
  "@type": "RadioChannel",
  "image": Object {
    "@type": "ImageObject",
    "height": 576,
    "url": "https://news.files.bbci.co.uk/ws/img/logos/og/urdu.png",
    "width": 1024,
  },
  "mainEntityOfPage": Object {
    "@id": "http://localhost:7080/urdu/bbc_urdu_radio/liveradio",
    "@type": "WebPage",
    "name": "بی بی سی اردو ریڈیو",
  },
  "publisher": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/urdu.png",
      "width": 1024,
    },
    "name": "BBC News اردو",
    "publishingPrinciples": "https://www.bbc.com/urdu/institutional-49282935",
  },
  "thumbnailUrl": "https://news.files.bbci.co.uk/ws/img/logos/og/urdu.png",
  "url": "http://localhost:7080/urdu/bbc_urdu_radio/liveradio",
}
`;

exports[`Canonical urdu liveRadio SEO OG description 1`] = `"اردو زبان کے معتبر ترین خبررساں اداروں میں ایک، بی بی سی اردو کی جانب سے مصدقہ خبریں اور جامع تجزیے"`;

exports[`Canonical urdu liveRadio SEO OG image 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/urdu.png"`;

exports[`Canonical urdu liveRadio SEO OG image alt 1`] = `"BBC News اردو"`;

exports[`Canonical urdu liveRadio SEO OG locale 1`] = `"ur"`;

exports[`Canonical urdu liveRadio SEO OG site name 1`] = `"BBC News اردو"`;

exports[`Canonical urdu liveRadio SEO OG title 1`] = `"بی بی سی اردو ریڈیو - BBC News اردو"`;

exports[`Canonical urdu liveRadio SEO OG type 1`] = `"website"`;

exports[`Canonical urdu liveRadio SEO OG url 1`] = `"http://localhost:7080/urdu/bbc_urdu_radio/liveradio"`;

exports[`Canonical urdu liveRadio SEO Page title 1`] = `"بی بی سی اردو ریڈیو - BBC News اردو"`;

exports[`Canonical urdu liveRadio SEO Robots meta tag 1`] = `"noodp,noydir"`;

exports[`Canonical urdu liveRadio SEO Twitter card 1`] = `"summary_large_image"`;

exports[`Canonical urdu liveRadio SEO Twitter creator 1`] = `"@bbcurdu"`;

exports[`Canonical urdu liveRadio SEO Twitter description 1`] = `"اردو زبان کے معتبر ترین خبررساں اداروں میں ایک، بی بی سی اردو کی جانب سے مصدقہ خبریں اور جامع تجزیے"`;

exports[`Canonical urdu liveRadio SEO Twitter image alt 1`] = `"BBC News اردو"`;

exports[`Canonical urdu liveRadio SEO Twitter image src 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/urdu.png"`;

exports[`Canonical urdu liveRadio SEO Twitter site 1`] = `"@bbcurdu"`;

exports[`Canonical urdu liveRadio SEO Twitter title 1`] = `"بی بی سی اردو ریڈیو - BBC News اردو"`;
